Assets/PlatformConfigPC.asset

```diff
@@ -28,7 +28,8 @@ MonoBehaviour:
   EnableExportFbx: 1
   EnableExportGlb: 1
   EnableExportJson: 1
-  EnableExportUsd: 1
```

Comment:
Why was USD disabled?

Reply:
Probably superstition; I got a strange error from the build system about USD on the next-to-last commit. Put it back now.